Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WMRDevice.cs

```diff
@@ -0,0 +1,437 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem;
+using Microsoft.MixedReality.Toolkit.Internal.Extensions;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.WSA.Input;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Devices.WindowsMixedReality
+{
+    // TODO - name not final
+    public class WMRDevice : IMixedRealityDevice
```

Comment:
What about Windows mixed reality device manager?

Reply:
renamed to "WindowsMixedRealityDevice" (it's a device not a manager :D)